### PR TITLE
Use git describe in About Dialog

### LINKED ISSUE #460: Use git describe in About Dialog
- [x] I have read the specified guidelines for issues

#396 switched to using the output of `git describe` in versions and various strings, but the About Dialog was left untouched. This is what it shows now:

About Dialog

**Describe the solution you'd like**
We could use the variable `GIT_DESCRIBE`, defined in `/CMakeLists.txt`.

Also might be worth adding one more reference here, just under the logo:
About Dialog main page

**Additional context**
Don't worry about git commits vs. actual versions; `git describe --abbrev=7` shows just the tag if it's on it.

### PR DESCRIPTION
**Pull request by humrochagf**
Resolves: #460

Use AUDACITY_VERSION_STRING to be consistent with other references using
git describe, like AudacityApp.cpp CLI arg -v

Signed-off-by: Humberto Rocha <humrochagf@gmail.com>

### CODE REVIEW

File: src/BuildInfo.h [hunk 1]
```diff
@@ -92,7 +92,7 @@ class BuildInfo {
             static wxString NoRevisionText = XO("No revision identifier was provided").Translation();
 
             #ifdef REV_LONG
-                return wxString( "[[https://github.com/tenacityteam/tenacity/commit/" ) + REV_LONG + "|" + wxString( REV_LONG ).Left(6) + "]] of " +  getRevisionDateTime();
+                return wxString( "[[https://github.com/tenacityteam/tenacity/commit/" ) + REV_LONG + "|" + wxString( AUDACITY_VERSION_STRING ) + "]] of " +  getRevisionDateTime();
```
Comment (nbsp) [inline src/BuildInfo.h]: Just the version should be sufficient, I don't see a need for timestamping the commit.

**Review by nbsp — CHANGES_REQUESTED** [review]
I think it's best if we use `GIT_DESCRIBE` directly, and eventually maybe replace all references to `AUDACITY_VERSION_STRING` with it. See #433.

**Comment by n0toose** [comment]
By the way, thank you for helping us with this!

File: src/BuildInfo.h (hunk 1 above)
Comment (humrochagf) [reply src/BuildInfo.h]: Alright, I'll remove the timestamping

**Comment by humrochagf** [comment]
Oh, no problem I'm glad to be able to help :)

About my choice for the AUDACITY_VERSION_STRING was because since all other versions related info inside the code is using it as a reference to get the description, it would be easier for a refactor later to maintain consistency on the version by using the same macro to change without forgetting that for the about page it behaves differently.

But I'll check to add the GIT_DESCRIBE injected into the context by cmake so I can use it :)

**Comment by humrochagf** [comment]
Alright, changed to use `GIT_DESCRIBE` directly, I had to inject it into the macro file generated by `Version.cmake` file to have access to it inside the code.
Also, I kept the name `GIT_DESCRIBE` there, but it could name something else if you think it's better.

**Review by nbsp — APPROVED** [review]
Everything looks good to me :D

**Comment by n0toose** [comment]
We'll merge this change in a bit, thanks for your help again! ❇️

**Comment by n0toose** [comment]
**Merged in e0f34e7.** For future reference, allowing maintainers to modify your pull request will allow us to rebase your change on top of master before we merge the change - that way, your change will appear as "Merged" on your profile. It's mostly a cosmetic thing either way.

Thanks for your help once again! <3

**Comment by humrochagf** [comment]
Oh, that's odd, I remember seeing the checkbox to allow editing as checked. Well, maybe I misread the option or the force push after the amend changed something?
But no problem I'll keep an eye on it next time :smile:

**Pull request closed without merging**